iam.tf

```diff
@@ -75,3 +67,36 @@ resource "aws_iam_role_policy_attachment" "ssm" {
   policy_arn = aws_iam_policy.ssm[count.index].arn
   role       = aws_iam_role.lambda.name
 }
+
+data "aws_iam_policy_document" "logs" {
+  count = var.cloudwatch_logs_enabled ? 1 : 0
+
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+    ]
+
+    resources = [
+      aws_cloudwatch_log_group.lambda.arn,
```

Comment:
according to the aws [docs](https://docs.aws.amazon.com/lambda/latest/operatorguide/access-logs.html), we could even remove `aws_cloudwatch_log_group.lambda.arn` - I've tested this with one of our functions

Reply:
👍